### PR TITLE
Use correct feature naming for SQL Server Always On Availabiity Groups

### PR DESCRIPTION
**Pull request by MikeWasson**

### CODE REVIEW

**Comment by MikeWasson** [comment]
@telmosampaio Please review - thanks!

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference-architectures/virtual-machines-windows/images/multi-region-application-diagram.png](https://github.com/mspnp/architecture-center/blob/fix/sql-feature-naming/docs/reference-architectures/virtual-machines-windows/images/multi-region-application-diagram.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/virtual-machines-windows/multi-region-application?branch=pr-en-us-42) |
[docs/reference-architectures/virtual-machines-windows/multi-region-application.md](https://github.com/mspnp/architecture-center/blob/fix/sql-feature-naming/docs/reference-architectures/virtual-machines-windows/multi-region-application.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/virtual-machines-windows/multi-region-application?branch=pr-en-us-42) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C4%5C14%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201704141648033556-42%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference-architectures/virtual-machines-windows/multi-region-application.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ Depending on the cause of a failover, you might need to redeploy the resources w
 * Application subsystems are healthy. 
 * Functional testing. (For example, the database tier is reachable from the web tier.)
 
-### SQL Server Always On configuration
+### Configuring the SQL Server Always On Availability Group
```
Comment (telmosampaio) [inline docs/reference-architectures/virtual-machines-windows/multi-region-application.md]: lose the article (the), use the imperative (we are removing -ing from all our headings, according to Christopher)

File: docs/reference-architectures/virtual-machines-windows/multi-region-application.md [hunk 2]
```diff
@@ -141,16 +141,16 @@ Traffic Manager is a possible failure point in the system. If the Traffic Manage
...
 
    > [!WARNING]
    > With forced failover, there is a risk of data loss. Once the primary region is back online, take a snapshot of the database and use [tablediff] to find the differences.
    >
    >
-- Traffic Manager fails over to the secondary region, but the primary SQL replica is still available. For example, the front-end tier might fail, without affecting the SQL VMs. In that case, Internet traffic is routed to the secondary region, and that region can still connect to the primary SQL replica. However, there will be increased latency, because the SQL connections are going across regions. In this situation, you should perform a manual failover as follows:
+- Traffic Manager fails over to the secondary region, but the primary SQL Server replica is still available. For example, the front-end tier might fail, without affecting the SQL Server VMs. In that case, Internet traffic is routed to the secondary region, and that region can still connect to the primary SQL Server replica. However, there will be increased latency, because the SQL Server connections are going across regions. In this situation, you should perform a manual failover as follows:
```
Comment (telmosampaio) [inline docs/reference-architectures/virtual-machines-windows/multi-region-application.md]: Change SQL server replica to SQL server database replica. We are replicating databases, not the server. Look for all instances.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference-architectures/virtual-machines-windows/images/multi-region-application-diagram.png](https://github.com/mspnp/architecture-center/blob/fix/sql-feature-naming/docs/reference-architectures/virtual-machines-windows/images/multi-region-application-diagram.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/virtual-machines-windows/multi-region-application?branch=pr-en-us-42) |
[docs/reference-architectures/virtual-machines-windows/multi-region-application.md](https://github.com/mspnp/architecture-center/blob/fix/sql-feature-naming/docs/reference-architectures/virtual-machines-windows/multi-region-application.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/virtual-machines-windows/multi-region-application?branch=pr-en-us-42) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C4%5C17%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201704171756496485-42%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**